var/spack/repos/builtin/packages/placement-algorithm/package.py

```diff
@@ -0,0 +1,54 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PlacementAlgorithm(PythonPackage):
+    """Morphology placement algorithm"""
+
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/building/placementAlgorithm"
+    git      = "ssh://bbpcode.epfl.ch/building/placementAlgorithm"
+
+    version('develop', branch='master')
+    version('2.0.0', tag='placement-algorithm-v2.0.0', preferred=True)
+
+    variant('app', default=False, description='Applications')
+    variant('synthesis', default=False, description='Morphology synthesis')
```

Comment:
You are right, it's not used at the moment, but something to be added in a couple of weeks.
If you think it's better to postpone adding variant definition until it's actually used, I'll remove it for now.

Reply:
Sounds good. Sorry to be nit-picky, but do you actually need the distinction between `+app` and `~app`? It seems to me that the only differences are the dependencies, and I would hope that with our new setup these would not be such a pain. I'm in favor of simplicity here…

Reply:
Indeed, the dependencies are the only difference.
If a bunch of optional heavy dependencies is not a big deal in Spack world since they are instantiated anyway in some other context, I'm fine with removing the variant.